src/platform/posix/tcp_ip_channel.c

```diff
@@ -17,6 +17,10 @@
 
 #include "proto/message.pb.h"
 
+#ifdef MIN
```

Comment:
why this?

Reply:
Because sometimes `MIN` is already defined, it is the case for Zephyr, so then we undef to avoid errors